src/index.ts

```diff
@@ -11,7 +11,10 @@ import deepmerge from 'deepmerge';
 import { findWorkspaceDir } from '@pnpm/find-workspace-dir';
 import { findWorkspacePackages } from '@pnpm/workspace.find-packages';
 import { packlist } from '@pnpm/fs.packlist';
-import { PackageCache } from '@embroider/shared-internals';
+import { createRequire } from 'module';
+
+const require = createRequire(import.meta.url);
```

Comment:
This is the same technique that decorator-transforms is using here: https://github.com/ef4/decorator-transforms/pull/37/files

(sorry for the noise, that repo doesn't have formatting enforcement... yet?)

This is absolutely fine for a node tool, but it breaks browser compatibility (a concern of decorator-transforms)